client/src/components/chat/chat.tsx

```diff
@@ -191,6 +195,7 @@ export function HistoryChat(args: ScrollingQuestionsParams): JSX.Element {
                   setQuestionVisibilityPref(m.questionId, show);
                 }}
                 visibility={isQuestionsAnswersVisible(m.questionId)}
+                displayMentorNames={displayMentorNames || true}
```

Comment:
this will make `displayMentorNames` always true. If you want the default to be false, probably easier to invert the property and call it `hideMentorNames`, so that when it's not passed in it's naturally `falsy`

Reply:
I agree with "we ALWAYS show the mentor names (we show them for both video mentors and for chat mentors)", so I removed the displayMentorNames.